mlos_core/mlos_core/tests/optimizers/bayesian_optimizers_test.py

```diff
@@ -34,13 +34,10 @@ def test_context_not_implemented_warning(configuration_space: CS.ConfigurationSp
         optimization_targets=['score'],
         **kwargs
     )
-    suggestion = optimizer.suggest()
+    suggestion, _ = optimizer.suggest()
     scores = pd.DataFrame({'score': [1]})
     context = pd.DataFrame([["something"]])
 
-    with pytest.raises(UserWarning):
-        optimizer.register(suggestion, scores, context=context)
-
     with pytest.raises(UserWarning):
         optimizer.suggest(context=context)
```

Comment:
Hmm, so we can register context, but not suggest with it?

Reply:
It's more about not sending the warning not being being thrown anymore

Reply:
I understand what the code in this test is doing.  I'm asking why we aren't supporting suggestions with context yet when we're adding support for registering with context.  It makes it seem like a blackhole of information or only partial support somehow.

Reply:
What does context for a suggestion mean?

Reply:
We discussed offline and is documented in the suggestion in the README.md above now.